python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -1,21 +1,26 @@
 import json
 import os
 from datetime import datetime
-from pathlib import Path
```

Comment:
@utkarsharma2 I think this is a nice feature to debug locally (e.g. run the benchmark for a subset of files), but I believe the CI should be running the tests for a more stable version of the config.

My suggestion is:
* by default, we keep the behaviour of retrieving the configuration from the config.json file. It is available in the Docker image. In this case, we'd print saying we are using the built-in version of the config.
* If `ASTRO_BENCHMARK_CONFIG_PATH` is defined, we can retrieve the config from either the local or remote file path referenced by it, and we'd be printing the content of that configuration file / its location within the logs.

This would give us the flexibility to troubleshoot while also having some reproducibility and trackability.